src/index.css

```diff
@@ -1,3 +1,194 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@1,400;1,700&family=Outfit:wght@400;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@300;400;700&display=swap');
+
 * {
   box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: 'RobotoSlab', serif;
+  background-color: #fafafa;
+}
+
+.navbar {
+  display: flex;
+  align-items: center;
```

Comment:
- [ ] [OPTIONAL] This is great using flexbox to layout your content, congrats 👏 , But as said by the previous reviewer, you can still improve this code by using DRY (don't repeat yourself) but why? well, I realized that you have used the property `display: flex` at multiple places, please let's keep the code clean by extracting the property into a separate class that can be reused at multiple places, see the example below: 

- CSS file
```
.d-flex {
  display: flex;
  align-items: center;
}
```
- HTML file
```
<div class="d-flex">{your content}</div>
```

And remember you can do this for all the other properties that seem to be repeated multiple times, With this, I promise you'll be coding like a pro 😉